gateway/src/main/java/com/lyft/data/gateway/ActiveClusterMonitor.java

```diff
@@ -91,7 +90,7 @@ private void notifyForNoWorkers(ClusterStats clusterStats) {
     emailNotifier.sendNotification("Number of workers", clusterStats.toString());
   }
 
-  private ClusterStats getPrestoClusterStats(ProxyServerConfiguration backend) {
+  private ClusterStats getPrestoClusterStats(ProxyBackendConfiguration backend) {
```

Comment:
Why change the method signature ?

Reply:
`ProxyBackendConfiguration` is sub type of `ProxyServerConfiguration` and local to gateway project. Where `ProxyServerConfiguration` resides in `proxyserver` lib. 
So under gateway we should use `ProxyServerConfiguration` .